include/caf/leds.rst

```diff
@@ -69,54 +68,74 @@ To define PWM ports used by the application, complete the following steps:
 			ch2-pin = <7>;
 		};
 
-	Make sure to configure all PWM ports that are used by the application.
-	In particular, the number of PWM ports used must match the value set in :option:`CONFIG_CAF_LEDS_COUNT`.
+	Make sure to configure all PWM ports and channels that are used by the application.
 	For more help, see :ref:`zephyr:dt-guide`.
 
-2. Create a configuration file with the following array of arrays:
+2. Enable or create the ledpwm nodes in the devicetree file by setting the status to ``"okay"``. This can be done by using one of the following options:
```

Comment:
What is the correct name of the nodes? `ledpwm`, `pmwled`, or `pwmleds`? Or `pwm_led`?
How many nodes do I need to create? What do these nodes do?